bin/getTitle.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:http/http.dart' as http;
+import 'package:html/parser.dart' show parse;
+// import 'package:html/dom.dart' hide Text;
+
+dynamic main() {
+  printTitle();
+}
+
+void printTitle() async{
+    var response = await http.get('https://example.com/');
+    // If the http request is successful the statusCode will be 200
+    if (response.statusCode == 200){
+      var document = parse(response.body);
+      
+      document.getElementsByTagName('title').forEach((child) {
```

Comment:
Two remarks here. `getElementsByTagName` returns a `List<Element>`. So it forces you to use `forEach` or a `for` loop. Since we know there will be only one <title> in the HTML, it's more convenient to use `querySelector`. See Document's [list of methods](https://pub.dev/documentation/html/latest/dom/Document-class.html#instance-methods).

Also, innerHTML is useful to find a node's HTML code. Since we want to find <title>'s text, how about using the `text` getter? See Element's [getters](https://pub.dev/documentation/html/latest/dom/Element-class.html#instance-properties).

```dart
final title = document.querySelector('title');
return title.text;
```